** extraction/compile.mli.orig	2022-11-25 16:05:15.794805523 +0100
--- extraction/compile.mli	2022-11-25 16:05:35.902601278 +0100
*************** open EProgram
*** 8,15 ****
  open Kernames
  open List0
  open MCList
- open Malfunction
  open Nat0
  open PCUICPrimitive
  open Specif
  open String0
--- 8,15 ----
  open Kernames
  open List0
  open MCList
  open Nat0
+ open Malfunction
  open PCUICPrimitive
  open Specif
  open String0