*** extraction/serialize.ml.orig	2023-07-11 16:34:04.052701333 +0200
--- extraction/serialize.ml	2023-07-11 16:34:07.019337982 +0200
*************** open CeresSerialize
*** 6,15 ****
  open CeresString
  open Datatypes
  open List0
! open Malfunction
  open Nat0
  open PeanoNat
- open PrimFloat
  open ReflectEq
  open String0
  open Bytestring
--- 6,15 ----
  open CeresString
  open Datatypes
  open List0
! open PrimFloat
  open Nat0
+ open Malfunction
  open PeanoNat
  open ReflectEq
  open String0
  open Bytestring
