*** extraction/serialize.mli.orig	2023-10-12 14:45:51.196806225 +0200
--- extraction/serialize.mli	2023-10-12 14:46:12.958768698 +0200
*************** open CeresSerialize
*** 8,22 ****
  open CeresString
  open Datatypes
  open List0
- open Malfunction
- open Nat0
- open PeanoNat
  open PrimFloat
  open PrimInt63
  open ReflectEq
  open String0
  open Uint0
  open Bytestring
  
  val _escape_ident : String.t -> String.t -> String.t
  
--- 8,22 ----
  open CeresString
  open Datatypes
  open List0
  open PrimFloat
  open PrimInt63
+ open Nat0
+ open PeanoNat
  open ReflectEq
  open String0
  open Uint0
  open Bytestring
+ open Malfunction
  
  val _escape_ident : String.t -> String.t -> String.t
  
