*** extraction/compile.mli.orig	2023-06-02 15:44:54.576915233 +0200
--- extraction/compile.mli	2023-06-02 15:45:19.446502642 +0200
*************** open EProgram
*** 8,15 ****
  open Kernames
  open List0
  open MCList
- open Malfunction
  open Nat0
  open Specif
  open Uint0
  open Utils_array
--- 8,15 ----
  open Kernames
  open List0
  open MCList
  open Nat0
+ open Malfunction
  open Specif
  open Uint0
  open Utils_array
