--- extraction/serialize.mli.orig	2024-02-16 17:29:25
+++ extraction/serialize.mli	2024-02-16 17:29:37
@@ -10,7 +10,6 @@
 open Kernames
 open List0
 open MCString
-open Malfunction
 open Nat0
 open PeanoNat
 open PrimFloat
@@ -19,6 +18,7 @@
 open String0
 open Uint0
 open Bytestring
+open Malfunction
 
 val _escape_ident : String.t -> String.t -> String.t
 
--- extraction/serialize.ml.orig	2024-02-16 17:28:37
+++ extraction/serialize.ml	2024-02-16 17:29:00
@@ -10,7 +10,6 @@
 open Kernames
 open List0
 open MCString
-open Malfunction
 open Nat0
 open PeanoNat
 open PrimFloat
@@ -19,6 +18,7 @@
 open String0
 open Uint0
 open Bytestring
+open Malfunction
 
 (** val _escape_ident : String.t -> String.t -> String.t **)
 
--- extraction/compile.mli.orig	2024-02-17 06:54:01
+++ extraction/compile.mli	2024-02-17 06:54:12
@@ -8,8 +8,8 @@
 open Kernames
 open List0
 open MCList
-open Malfunction
 open Nat0
+open Malfunction
 open Specif
 open Bytestring
 open Utils_array
