*** extraction/serialize.ml.orig	2023-10-30 16:28:30.018617698 +0100
--- extraction/serialize.ml	2023-10-30 16:28:57.148071741 +0100
*************** open CeresSerialize
*** 8,22 ****
  open CeresString
  open Datatypes
  open List0
- open Malfunction
- open Nat0
- open PeanoNat
  open PrimFloat
  open PrimInt63
  open ReflectEq
  open String0
  open Uint0
  open Bytestring
  
  (** val _escape_ident : String.t -> String.t -> String.t **)
  
--- 8,22 ----
  open CeresString
  open Datatypes
  open List0
  open PrimFloat
  open PrimInt63
+ open Nat0
+ open PeanoNat
  open ReflectEq
  open String0
  open Uint0
  open Bytestring
+ open Malfunction
  
  (** val _escape_ident : String.t -> String.t -> String.t **)
  
